### DISCUSSION TITLE
DBeaver does not show error unless it is forced to.

### DISCUSSION THREAD

**Discussion by remuslupin** [dbeaver/dbeaver#20852] [Q&A]
I have a query that returns 100 rows when I hit Ctrl+A, and it returns 263 rows when I hit Ctrl+Alt+Shift+A.
When I add a calculated column like 1 / column_x and I am pretty sure that some of the values in column_x are 0, DBeaver only returns 13 rows. When I add column_x=0 in WHERE clause, then I get the error "SQL Error [8134] [S0001]: Divide by zero error encountered.

Error position:"

Could anyone let me know if all platforms like that, or there is something that I am missing?

**Comment by E1izabeth** [comment]
We haven't meet such behavior yet. Could you figure out and share some minimal example for us to reproduce your issue?
